src/store/actions/contactInfo.js

```diff
@@ -1,9 +1,23 @@
 import { ACTION_TYPES } from '../actionTypes';
 
+const dataSanitization = data => {
```

Comment:
this function is not an action, correct? then I don't think it belongs to this file, this file should contain actions only, maybe you can agree on a place where you can place your utility functions